docs/resources/elasticsearch_security_api_key.md

```diff
@@ -41,6 +41,37 @@ resource "elasticstack_elasticsearch_security_api_key" "api_key" {
   })
 }
 
+# restriction on a role descriptor for an API key is supported since Elastic 8.9
```

Comment:
This is failing lint because it's not consistent with the result of docs generation. Are you able to run `make docs-generate` locally and get things trued up?

Reply:
For sure :+1: